portfolio/src/main/webapp/css/custom.css

```diff
@@ -163,3 +163,25 @@ body {
 .dropdown:hover .dropbtn {
   background-color: #6ab5cc;
 }
+
+#map-container {
+  width: 100%;
+  margin: auto;
+}
```

Comment:
Can you add lines between these for consistency.

Reply:
done!